app/helpers/questions_helper.rb

```diff
@@ -44,7 +44,7 @@ def format_questions_field(q, field)
     when "actions" then table_action_links(q)
     when "name"
       if params[:controller] == 'forms'
-        q.name + render_tags(q.sorted_tags)
+        (q.name + render_tags(q.sorted_tags)).html_safe
```

Comment:
I would rather not do this as it's not super safe.

Can't you just do `h(q.name) << render_tags(q.sorted_tags)`?

`render_tags` should be returning html_safe output. q.name is not html_safe, so it is causing the problem when concatenated. But if you do as above, both operands should be safe, and we should be good.